README.md

```diff
@@ -568,45 +556,6 @@ const result = graphQlQueryToJson(query, {
 - Error if variables are referenced but not provided
 - Error if query contains multiple operations
 
-### Helper Functions
-
-The library also exports utility functions:
-
-```ts
-import { flatMap, isString, isArray, isObject } from 'graphql-query-to-json'
-
-// Custom array flattening
-flatMap([1, 2, 3], x => [x, x + 1]) // [1, 2, 2, 3, 3, 4]
-
-// Type checking utilities
-isString("hello") // true
-isArray([1, 2, 3]) // true
-isObject({}) // true
-```
-
-## Error Handling
```

Comment:
My thought was that it only adds a bit of bloat to the readme and it is also very easy for this to go out of sync with the real library.

Reply:
Agree